.github/workflows/linting.yml

```diff
@@ -62,6 +62,7 @@ jobs:
         with:
           config-file: ".markdownlinkcheck.json"
           check-modified-files-only: "yes"
+          base-branch: "release-0.8"
```

Comment:
I wonder if we can grab this dynamically from the github context.

Reply:
https://stackoverflow.com/questions/58033366/how-to-get-current-branch-within-github-actions

May be we can capture this env var and then use it here?, at least as a patch for the master branch, this one seems fine now.

Reply:
Ah yes, `$GITHUB_BASE_REF`

Reply:
https://github.com/submariner-io/submariner/pull/1086